KarhooUISDK/Screens/PassengerDetails/PassengerDetailsViewController.swift

```diff
@@ -244,9 +244,13 @@ final class PassengerDetailsViewController: UIViewController, PassengerDetailsVi
     // MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        keyboardSizeProvider.register(listener: self)
         forceLightMode()
     }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        keyboardSizeProvider.register(listener: self)
```

Comment:
If we're registering in `viewWillAppear` then we need to unregister in `viewWillDisappear` instead of `deinit`